spec/requests/api/v3/activity_resource_spec.rb

```diff
@@ -80,9 +84,40 @@
       end
     end
 
-    it_behaves_like 'handling anonymous user', 'Activity', '/api/v3/activities/%s' do
+    it_behaves_like 'handling anonymous user' do
       let(:project) { FactoryGirl.create(:project, is_public: true) }
-      let(:id) { activity.id }
+      let(:path) { api_v3_paths.activity activity.id }
+    end
+  end
+
+  describe '#patch' do
+    subject(:response) { last_response }
+    let(:patch_path) { api_v3_paths.activity activity.id }
```

Comment:
Do you think the patch_path and the get_path are likely to diverge in the near future? Because if not, then you should be able to move it into the initialization section in the beginning of this file. But if you say that this is about consistency across the recource specs I am fine with that too.